src/databricks/labs/remorph/transpiler/dialects_config.py

```diff
@@ -0,0 +1,34 @@
+from sqlglot.dialects.dialect import Dialect, Dialects
+
+from databricks.labs.remorph.snow import databricks, experimental, snowflake
+
+
+class DialectConfig:
```

Comment:
it's not a config, it's a factory. but why do we need a whole dedicated file for it?..

Reply:
Refactored this inside Config